include/ccf/service/tables/jwt.h

```diff
@@ -58,18 +58,37 @@ namespace ccf
   using JwtKeyId = std::string;
   using Cert = std::vector<uint8_t>;
 
+  struct KeyMetadata
```

Comment:
I suggest renaming to JWKMetadata to make it obvious which Key this is about, because there are a fair few kinds of keys in the KV.